mlflow/tracking/fluent.py

```diff
@@ -1028,6 +1028,20 @@ def _get_experiment_id():
 
 
 def autolog(log_input_example=False, log_model_signature=True):  # pylint: disable=unused-argument
+    """
+    Enable autologging for all supported integrations.
+
+    The parameters are passed to any autologging integrations that support them.
+
+    See the :ref:`tracking docs <automatic-logging>` for a list of supported autologging
+    integrations.
+
+    :param log_input_example: if True, logs a sample of the training data as part of the model
+                              as an example for future reference. If False, no sample is logged.
```

Comment:
Autologging produces multiple models over multiple training sessions. Accordingly, I think we should say something like `if True, input examples from training datasets will be collected and logged along with model artifacts during training`

I think we may want to pluralize these configuration parameters - e.g., `log_input_example -> log_input_examples`. If that makes sense to you, I'll file a follow-up PR for it.

Reply:
ah yeah, that makes perfect sense for the universal autolog.

Could see it also being pluralized for the individual autologs.